src/main/java/edu/studyup/serviceImpl/EventServiceImpl.java

```diff
@@ -29,28 +29,31 @@ public Event updateEventName(int eventID, String name) throws StudyUpException {
 		return event;
 	}
 
+	//BUG #2: Inefficient use of keySet iterator
```

Comment:
BUG #2: I totally agree with you that we should use the entrySet() iterator since it is more efficient than the previous one.